File: refactored/assertion-roulette/CSharpXUnit/CSharpXUnit.Tests/Tests/TimeSpanArgumentHasDefault.cs
using Xunit;

public class ArgumentTests
{
    [Fact]
    public void TimeSpanArgumentHasDefault()
    {
        {
            var arg = new SampleTimeSpanArgument(TimeSpan.FromMinutes(3));
            var command = UnitTestCommand.FromArgument(arg);
            int exitCode = command.Invoke(new string[0]);
            Assert.Equal(0, exitCode, "Explanation message");
            Assert.True(command.IsCommandRun(), "Explanation message");
            Assert.Equal(TimeSpan.FromMinutes(3), arg.GetValue(), "Explanation message");
        }
    }
}

File: refactored/assertion-roulette/java-junit/TimeSpanArgumentHasDefault.java
import org.junit.Test;
import static org.junit.Assert.*;

public class ArgumentTests {

    @Test
    public void timeSpanArgumentHasDefault() {
        {
            SampleTimeSpanArgument arg = new SampleTimeSpanArgument(Duration.ofMinutes(3));
            UnitTestCommand command = UnitTestCommand.fromArgument(arg);
            int exitCode = command.invoke(new String[0]);
            assertEquals("Explanation message", 0, exitCode);
            assertTrue("Explanation message", command.isCommandRun());
            assertEquals("Explanation message", Duration.ofMinutes(3), arg.getValue());
        }
    }
}
